cmd/dep/status.go

```diff
@@ -246,6 +378,21 @@ func runStatusAll(out outputter, p *dep.Project, sm *gps.SourceMgr) error {
 				PackageCount: len(proj.Packages()),
 			}
 
+			// Get children only for specific outputers
+			// in order to avoid slower status process
+			switch out.(type) {
+			case *dotOutput:
+				r := filepath.Join(p.AbsRoot, "vendor", string(proj.Ident().ProjectRoot))
+				ptr, err := gps.ListPackages(r, string(proj.Ident().ProjectRoot))
```

Comment:
Also, side note - by placing this within the branch of the if where the memo matches, we create a potentially confusing situation for users where they ask for dot output, but just don't get if the memos don't match. I'm undecided if it's out of scope to handle in this PR, but if we don't, then we need a follow-up issue.